src/GeneralStateTestsFiller/stBadOpcode/undefinedOpcodeFirstByteFiller.yml

```diff
@@ -0,0 +1,760 @@
+# Checks that behaviour of already deployed code, having invalid opcodes as the first byte.
```

Comment:
This could be merged separately already, becaue EIP-354 doesn't change this behavior.